src/requests/RequestApi.js

```diff
@@ -115,10 +115,28 @@ export class RequestApi extends Request {
               },
               done: () => {
                 if (append.id) console.log(append.id);
-  
-                this._currentReq[hash].object.d = this._currentReq[
-                  hash
-                ].object.d.map((e) => Object.values(e));
+                if (this.type === 'json') {
```

Comment:
we're missing the default `json/compact` case in this conditions. If we're no using it we could remove from the prop default values 👀

Reply:
Alternatively, if all `json` cases are similar, we could use a regex for the `if` statement, instead of a `===`. 

Something like: 
```
let jsonRegex = new RegExp("json", "i");
let isJsonType = jsonRegex.test(this.type);

if (isJsonType) {
....
```

Reply:
Cases that don't require specific transformations are managed in the else

Reply:
ok ✅